spanner/integration_test.go

```diff
@@ -3264,6 +3266,85 @@ func TestIntegration_DirectPathFallback(t *testing.T) {
 	}
 }
 
+func TestIntegration_GFE_Latency(t *testing.T) {
+	t.Parallel()
+	ctx, cancel := context.WithTimeout(context.Background(), 5*time.Minute)
+	defer cancel()
+
+	te := testutil.NewTestExporter(GFEHeaderMissingCountView, GFELatencyView)
+	GFELatencyOrHeaderMissingCountEnabled = true
```

Comment:
This is the variable that users must set to `true` to enable this feature, right? If so, I think we should rename the variable to simply `GFELatencyMetricsEnabled`. The fact that we also count the number of times that the header is missing, is a technical detail that users do not need to know about.